pkg/granted/registry/add.go

```diff
@@ -0,0 +1,94 @@
+package registry
+
+import (
+	"errors"
+	"fmt"
+	"net/url"
+	"os"
+	"os/exec"
+	"strings"
+
+	"github.com/urfave/cli/v2"
+)
+
+var AddCommand = cli.Command{
+	Name: "add",
+	Action: func(c *cli.Context) error {
+
+		if c.Args().Len() != 1 {
```

Comment:
Make sure to test with SSH as well as HTTPS - you should still be able to use the `net/url` library to parse these. e.g. `git@github.com:common-fate/granted-registry.git`

Reply:
Was not able to use this for SSH. I have reverted to writing a custom regex parser for now. Resolving this for now.